app/build.gradle.kts

```
@@ -0,0 +1,107 @@
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("com.google.gms.google-services")
+    id("com.google.firebase.crashlytics")
+    id("com.google.firebase.firebase-perf")
+    id("dagger.hilt.android.plugin")
+    id("com.ncorti.ktfmt.gradle") version "0.10.0"
+}
+
+android {
+    compileSdk = 33
+    compileSdkPreview = "UpsideDownCake"
+    namespace = "com.example.makeitso"
+
+    defaultConfig {
+        applicationId = "com.example.makeitso"
+        minSdk = 21
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0"
+        targetSdkPreview = "UpsideDownCake"
+
+        testInstrumentationRunner = "com.example.makeitso.MakeItSoTestRunner"
+        vectorDrawables.useSupportLibrary = true
+    }
+
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+        }
+    }
+    flavorDimensions.add("environment")
```

nit: You can use the `+=` operator instead.
```suggestion
    flavorDimensions.add("environment")
```

---

Done!